tests/misc_testsuite/tail-call/loop-across-modules.wast

```diff
@@ -42,5 +42,5 @@
   (start $start)
 )
 
-(assert_return (invoke $B "g" (i32.const 100000000))
+(assert_return (invoke $B "g" (i32.const 100000))
```

Comment:
I wasn't sure how best to handle this, basically the test takes forever on an unoptimized interpreter (aka `cargo test`) so I turned this down to be more reasonable, but that means this probably isn't testing what it originally wanted to test since the number is significantly smaller.

Reply:
I think this is probably fine as-is, since 100,000 is still pretty big, but we could add an import like `(import "env" "tail-recursion-count" (func (result i32))` and configure it appropriately in the wast utils crate.

Reply:
I'm a bit wary to extend the namespace of builtins just yet so I'm follow your gut and leave this at 100k for now. We might want a little more "official-ness" around extending the set of names `wast` can import by default than a one-off thing such as this.